app/subscriber/src/features/my-reports/components/ReportKindIcon.tsx

```diff
@@ -1,29 +1,58 @@
-import { FaUserClock } from 'react-icons/fa';
-import { FaClock, FaUser } from 'react-icons/fa6';
 import { getReportKind, IReportModel, ReportKindName } from 'tno-core';
 
 import * as styled from './styled';
 
 export interface IReportKindIconProps {
   report?: IReportModel;
+  includeDescription?: boolean;
 }
 
-export const ReportKindIcon = ({ report }: IReportKindIconProps) => {
+export const ReportKindIcon = ({ report, includeDescription }: IReportKindIconProps) => {
   if (!report) return null;
 
   const instance = report.instances.length ? report.instances[0] : undefined;
   const isActive = !instance?.sentOn;
   const kind = getReportKind(report);
 
-  let icon: React.ReactNode;
-  if (kind === ReportKindName.Auto) icon = <FaUserClock title="Auto Report" />;
-  else if (kind === ReportKindName.AutoSend) icon = <FaClock title="Auto Send Report" />;
-  else if (kind === ReportKindName.Manual) icon = <FaUser title="Manual Report" />;
-  else icon = <></>;
+  let reportType:
```

Comment:
I like these changes

Reply:
Be careful not to lose the status change I made on the ReportCard.tsx

Reply:
Just finished the rebase, kept it in there!